### PR TITLE
s4583972 Perceiver AKOA knee

### PR DESCRIPTION
**Pull request by humphreymunn**
Hi Shakes and COMP3710,

My pull request contains an implementation of the Perceiver task for knee laterality classification on the AKOA knee data. 

I achieved an accuracy of 91% on the test set. The details of the training are in the README file.

Files:

diagrams: folder for readme figures
README.md: documentation and report on perceiver, dataset, implementation and results
driver.py: main script, trains the model and evaluates it
perceiver_model.py: contains the TensorFlow model of the perceiver

Thank you for reviewing this request,

Humphrey Munn (45839724)

### CODE REVIEW

**Comment by SiyuLiu0329** [comment]
Difficulty Hard

Functionality: OK


Code
- Structure:  OK 
- Code commenting could use some improvement

README 
- Content: OK 
- Please also references any guides you have used (if any)

PR Target OK

General Comments
- Commit messages:  OK

**Comment by humphreymunn** [comment]
Thanks Siyu for the comment. I haven't used any guides, except for the Fourier encoding part which I have referenced.

**Comment by shakes76** [comment]
## TF/Torch Usage
Adequate use and implementation
## Good Practice (Design/Commenting)
Good spacing and comments
Header blocks
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage 
Module present
No Data leakage
Difficulty: Hard
## Commit Log
Meaningful commit messages
Progressive commits used
## Documentation
Good Description and Comments 
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
No Feedback required
Request Description good
